cadquery/cq.py

```diff
@@ -4120,6 +4120,18 @@ def cylinder(
         If combine is false, the result will be a list of the cylinders produced.
         """
 
+        if isinstance(centered, bool):
+            centered = (centered, centered, centered)
+
+        xyz = [
```

Comment:
Local variable 'xyz' is assigned to but never used.

Reply:
Thank you!